tests/migration_test.go

```diff
@@ -2065,39 +2077,45 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				By("Waiting for VMI to disappear")
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 			},
-				table.Entry("[sig-storage][test_id:2226] with ContainerDisk", newVirtualMachineInstanceWithFedoraContainerDisk),
-				table.Entry("[sig-storage][rook-ceph][test_id:2731] with OCS disk from block volume PVC", newVirtualMachineInstanceWithFedoraOCSDisk),
-			)
-			It("[sig-compute][test_id:3241]should be able to cancel a migration right after posting it", func() {
-				vmi := tests.NewRandomFedoraVMIWithGuestAgent()
-				vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse(fedoraVMSize)
+				table.Entry("[sig-storage][test_id:2226] with ContainerDisk", newVirtualMachineInstanceWithFedoraContainerDisk, false),
+				table.Entry("[sig-storage][rook-ceph][test_id:2731] with OCS disk from block volume PVC", newVirtualMachineInstanceWithFedoraOCSDisk, false),
+				table.Entry("[sig-storage][test_id:2228] with ContainerDisk", newVirtualMachineInstanceWithFedoraContainerDisk, true),
+				table.Entry("[sig-storage][rook-ceph][test_id:2732] with OCS disk from block volume PVC", newVirtualMachineInstanceWithFedoraOCSDisk, true))
 
-				By("Starting the VirtualMachineInstance")
-				vmi = runVMIAndExpectLaunch(vmi, 240)
+			testMigrationCancelletion := func(ithdr string, with_virtctl bool) {
```

Comment:
can you use a table here instead of defining the test function with an `It()` in it like this?  I'm only making this request because it sticks with our current conventions.

Reply:
Thanks for reviewing. I'll change it.

Reply:
Done in v4